src/models/chains.rs

```diff
@@ -18,3 +20,17 @@ pub struct NativeCurrency {
     pub symbol: String,
     pub decimals: u64,
 }
+
+#[derive(Serialize, Deserialize, Debug, PartialEq, Clone)]
+#[serde(rename_all = "camelCase")]
+pub struct Theme {
+    pub text_color: String,
+    pub background_color: String,
+}
+
+#[derive(Serialize, Deserialize, Debug, PartialEq, Clone)]
+#[serde(rename_all = "camelCase")]
+pub struct GasPriceOracle {
+    pub url: String,
+    pub gas_parameter: Option<String>,
```

Comment:
Is this the only format we will have? 

Afaik the web version also supports having a constant value set. And there was the discussion about a multiplier for the oracle solution.

Reply:
If that is the situation we should probably change this from a `struct` into a `enum` and make it polymorphic.

Reply:
The `gas_parameter` can have any `String` value (it'd depend on the oracle itself)